README.md

```diff
@@ -67,6 +67,7 @@ Any `processingDate` body parameter in responses should be replaced with the `{{
 
 Also remember to check `app/uk/gov/hmrc/uknwauthcheckerapistub/utils/Constants.scala` for the list of valid Eoris, any Eori that is not on that list will return `valid: false` and `code: 1`, while the listed Eoris will return `valid: true` and `code: 0` 
 
+Any duplicate eori numbers in the body of the request will be flattened and the response will only contain one instance of the eori.
 ### License
```

Comment:
Please can we have a gap between this and the license heading below.

Also can you add a heading for the duplicate eoris, so it stands out.